content/resources/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: resources
```

Comment:
Should this be in title case?

Reply:
hm no, well in fact i wanted them all to be lower case (looks more cool :-))

but i just now noticed that "Welcome" is upper case ... lowercased that one then :-)